src/PaymentPart/Output/MarkupOutput/Template/Html/PlaceholderElementTemplate.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\MarkupOutput\Template\Html;
+
+class PlaceholderElementTemplate
+{
+    public const TEMPLATE = <<<EOT
+<img src="{{ placeholder-file }}" style="width:{{ placeholder-width }}mm; height:{{ placeholder-height }}mm; float:{{ placeholder-float }}" class="qr-bill-placeholder" id="{{ placeholder-id }}">
```

Comment:
The `float` style was not here before. Is it required? Does it fix anything in `HtmlOutput`?

Reply:
Due to the specs (https://www.paymentstandards.ch/dam/downloads/style-guide-en.pdf page 7) it seems to me that the two ploaceholders at the bottom are clearly `float: right` elements. You have achieved this partially by using ids for the different placeholders and then setting `float: right` where needed by css.
In xml the are no stylesheets, so the individual differences of each placeholders had to be predefined like some other characteristics that were already set in \Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder. 

I consider the margin-top and also the floating properties to be predefined by the spec. So they cannot vary, same as width and height. Placing those parameters in the \Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder class solves this problem and makes post-styling unnecessary.